katas/adriel-gimenes/01-30/RGB-To-Hex-Conversion/RgbToHexConversion.cs

```diff
@@ -0,0 +1,16 @@
+namespace katas.AdrielGimenes;
+
+public class RgbToHexConversion
+{
+    public static string Rgb(int r, int g, int b)
+    {
+        return ToHex(r) + ToHex(g) + ToHex(b);
+    }
+
+    private static string ToHex(int value)
+    {
+        if (value < 0) return "00";
+        if (value > 255) return "FF";
+        return value.ToString("x2").ToUpper().PadLeft(2, '0');
```

Comment:
Nice C# formatting option used here, although it would have been nicer to practice your logic and problem solving skills by implementing a method of your own to convert decimal to hexadecimal numbers

Reply:
Noted.